code/modules/modular_computers/hardware/battery_module.dm

```diff
@@ -14,7 +14,7 @@
 
 /obj/item/computer_hardware/battery/handle_atom_del(atom/A)
 	if(A == battery)
-		try_eject(0, null, TRUE)
+		QDEL_NULL(battery)
```

Comment:
Isn't this called when `battery` is already being deleted?

Reply:
battery is the battery inside of this item, which is actually a module for holding the batteries.

Reply:
guess where `handle_atom_del` is called from

Reply:
hnng good point I'm brain farting. hold on I'll fix this